dynamic-plugins/imports/package.json

```
@@ -27,8 +27,8 @@
     "@janus-idp/backstage-plugin-quay": "1.11.3",
     "@janus-idp/backstage-plugin-rbac": "1.29.4",
     "@janus-idp/backstage-plugin-tekton": "3.12.3",
-    "@janus-idp/backstage-plugin-bulk-import": "1.4.6",
-    "@janus-idp/backstage-plugin-bulk-import-backend": "1.4.1",
+    "@janus-idp/backstage-plugin-bulk-import": "1.4.7",
+    "@janus-idp/backstage-plugin-bulk-import-backend": "1.5.1",
```

```suggestion
    "@janus-idp/backstage-plugin-bulk-import-backend": "1.5.2",
```

Bumping to cherry-pick https://github.com/janus-idp/backstage-showcase/pull/1618, because the automated cherry-picked could not be performed: https://github.com/janus-idp/backstage-showcase/pull/1618#issuecomment-2349373114
(conflicts expected because https://github.com/janus-idp/backstage-showcase/pull/1612 should have been merged first)